scripts/xapi.conf

```diff
@@ -375,6 +375,9 @@ sm-plugins=ext nfs iscsi lvmoiscsi dummy file hba rawhba udev iso lvm lvmohba lv
 # migration-compression = true
 # if not requested otherwise, use stream compression during migration of a VM
 
+# evacuation-batch-size = 10
```

Comment:
xenopsd has 16 workers (to match the 16 vCPUs of Dom0), so we could make this default to 16. For now 10 is probably fine though, we should have some tests that measure what the optimal number would be here (a too high number might increase VM downtime if the memory is not compressible well, or if you have vGPUs)